astrocut/cube_cut.py

```diff
@@ -84,27 +89,34 @@ def __init__(self):
                          "VIGNAPP": [None, "vignetting or collimator correction applied"]}
 
         
-    def _parse_table_info(self, table_data, verbose=False):
+    def _parse_table_info(self, product, table_data, verbose=False):
```

Comment:
we do use this function in tesscut. perhaps we shouldn't be, since it's "internal". or perhaps it should not be an internal function. nevertheless, it's fine as you have it, but it might have been nice to have had product be a keyword argument that defaulted to SPOC instead of being a positional arg so that downstream code wouldn't have to change. I believe you are also taking care of the downstream code, at least in tesscut. Are there any other packages that might use this function? Astroquery?

Reply:
i'm fine with not changing this now. just pointing it out that we need to change it in tesscut (also in the s3_support branch, when we switch over to using this)

Reply:
I'm in favor of changing this here to prevent any future bugs elsewhere. I'll work on this next.